### PR TITLE
Update NOTICE.txt

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -113,6 +148,19 @@ SOFTWARE.
 
 ---
 
+## @sentry/react-native
```
Comment (amyblais) [inline NOTICE.txt]: Was added via https://github.com/mattermost/mattermost-mobile/commit/1efb01deda81cc0ee0fa0a36af8c02e342272f11#diff-b9cfc7f2cdf78a7f4b91a753d10865a2.

Comment (amyblais) [reply NOTICE.txt]: @enahum If you can provide a link for this and then I can fill in the details.

File: NOTICE.txt [hunk 2]
```diff
@@ -2067,41 +2183,6 @@ limitations under the License.
...
 ---
 
-## react-native-sentry
-
-This product contains 'react-native-sentry' by Sentry.
-
-Official Sentry SDK for react-native
-
-* HOMEPAGE:
-  * https://github.com/getsentry/react-native-sentry
```
Comment (enahum) [inline NOTICE.txt]: @amyblais this is the link for sentry

**Comment by amyblais** [comment]
@enahum Please help merge + cherry-pick.

**Pull request merged**